public/index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    	<link href='http://fonts.googleapis.com/css?family=Montserrat:400,700' rel='stylesheet' type='text/css'>
+
+    	<link rel="stylesheet" href="app.css">
+    <title>Photography Website
+    </title>
+</head>
+<body>
+	<div class="top">
+		<span class="icons">
+			<i class="fa fa-arrow-left"></i>
+				/
+			<i class="fa fa-tint"></i>
+		</span>
+		<span class="name1">
+			Jacklyn Lee
+		</span>
+		<span class="name2">
+			Photography
+		</span>
+		<span class="links">
+			<a href="#">About</a>
+			<a href="#">Work</a>
+			<a href="#">Contact</a>
+		</span>
+	</div>
+	<div class="logo">
+		<img src="images/logo.png" alt="logo">
+		<span class="title">
+			Art Photos
+		</span>
+		<span class="chevron">
+			<i class="fa fa-chevron-down"></i>
+		</span>
+	</div>
+	<div class="row">
+		<div class="photo">
+			<img src="images/1.png" alt="ceiling">
+			<span class="phototitle">Hoodie stumptown kitsch</span>
+			<span class="date">05/05/2015</span>
+		</div>
+		<div class="photo-right">
+			<ul class="description">
+				<li>
+					<i class="fa fa-camera"></i>
+					Canon PowerShot S95
+				</li>
+				<li>
+					<i class="fa fa-eye"></i>
+					22.5mm
+				</li>
+				<li>
+					<i class="fa fa-adjust"></i>
+					f/5.6
+				</li>
+				<li>
+					<i class="fa fa-clock-o"></i>
+					80
+				</li>
+			</ul>
+		</div>
+	</div>
+	<div class="row">
+		<div class="photo">
+			<img src="images/2.png" alt="ceiling">
+			<span class="phototitle">Hoodie stumptown kitsch</span>
+			<span class="date">05/05/2015</span>
+		</div>
+		<div class="photo-right">
+			<ul class="description">
+				<li>
+					<i class="fa fa-camera"></i>
+					Canon PowerShot S95
+				</li>
+				<li>
+					<i class="fa fa-eye"></i>
+					22.5mm
+				</li>
+				<li>
+					<i class="fa fa-adjust"></i>
+					f/5.6
+				</li>
+				<li>
+					<i class="fa fa-clock-o"></i>
+					80
+				</li>
+			</ul>
+		</div>
+	</div>
+	<div class="row">
+		<div class="photo">
+			<img src="images/3.png" alt="ceiling">
+			<span class="phototitle">Hoodie stumptown kitsch</span>
+			<span class="date">05/05/2015</span>
+		</div>
+		<div class="photo-right">
+			<ul class="description">
+				<li>
+					<i class="fa fa-camera"></i>
+					Canon PowerShot S95
+				</li>
+				<li>
+					<i class="fa fa-eye"></i>
+					22.5mm
+				</li>
+				<li>
+					<i class="fa fa-adjust"></i>
+					f/5.6
+				</li>
+				<li>
+					<i class="fa fa-clock-o"></i>
+					80
+				</li>
+			</ul>
+		</div>
+	</div>
+	<div class="bottom">
+		<h2 class="bottom-text">
+			If you enjoyed this demo you might also like:
+		</h2>
+		<img src="images/5.png">
```

Comment:
Group the `img` and its related text together. This will make replication easier as well simplifying styles.